HotelBooking/Views/utils.py

```diff
@@ -13,3 +13,10 @@ def medium_print(text: str):
 
 def error_print(prefix: str = "", suffix: str = ""):
     print(f'\n{prefix}Please try again{suffix}')
+
+def ValidateDate(date_text):
+        try:
+            datetime.strptime(date_text, '%m/%d/%Y')
```

Comment:
What's the rationale behind testing it as mm/dd/yy? Checking the SQL file it appears we have a different date format?

https://github.com/mclhtay/CS4471/blob/main/HotelBooking/Database/create_database.sql#L89